backend/src/packages/chaiNNer_standard/image_filter/correction/color_transfer.py

```diff
@@ -19,42 +41,55 @@
             best results.""",
     icon="MdInput",
     inputs=[
-        ImageInput("Image", channels=[1, 3, 4]),
+        ImageInput("Image", channels=[3, 4]),
```

Comment:
That is a breaking change. Why did you remove support for grayscale images?

Reply:
As I mentionned it above, there was a bug:
![image](https://github.com/chaiNNer-org/chaiNNer/assets/137266603/0b23ed91-9228-45da-8056-ebf0d1025e00)

Reply:
You could have it convert to RGB automatically instead

Reply:
I could.
But results are so bad that I think that it will be a bad user experience. And this does not work for the 2 added algos.
This is to transfer colors, not to colorize grayscale images.
![chaiNNer-color-transfer-2023-7-1_22-32](https://github.com/chaiNNer-org/chaiNNer/assets/137266603/45fa45f3-04ab-4a89-9fe6-6a14a6289d91)

Reply:
"But results are so bad"  <- No, this is completely logical that it does not work.
Color transfer means that the color information is still here before the transfer. The 'color' information is lost during the BGR->grayscale conversion.

Reply:
True. In that case, we can either do a migration or just not do that and hope nobody was doing gray with this before. I doubt anyone was, and if it doesn't work with it gray then it's not really a loss at all

Reply:
I would say that we just declare supporting grayscale images a bug. Colorization isn't the primary purpose of this node anyway.